README.md

```diff
@@ -23,16 +35,29 @@ The pyroSAR package aims at providing a complete solution for the scalable organ
 This software used to run on a local server and is currently being restructured into a Python package.
 Not everything is working properly, stay tuned...
 
+# Installation
+For the installation we need the Python tool pip and the version control system git. On Windows pip is 
```

Comment:
Since pyroSAR is now registered in [PyPi](https://pypi.org/project/pyroSAR/) we don't need this long installation procedure. 
We just need to do pip install pyroSAR